govcd/disk.go

```diff
@@ -124,8 +128,12 @@ func (d *Disk) Update(newDiskInfo *types.Disk) (Task, error) {
 	}
 
 	// Send request
-	req := d.client.NewRequest(nil, http.MethodPut, *reqUrl, bytes.NewBufferString(xml.Header+string(xmlPayload)))
-	req.Header.Add("Content-Type", execLink.Type)
+	reqPayload := bytes.NewBufferString(xml.Header + string(xmlPayload))
+	req := d.client.NewRequest(nil, http.MethodPut, *reqUrl, reqPayload)
+	req.Header.Add("Content-Type", updateDiskLink.Type)
+	util.Logger.Printf("[DEBUG] POSTING TO URL: %s", reqUrl)
+	util.Logger.Printf("[DEBUG] XML TO SEND:\n%s", reqPayload)
```

Comment:
All information regarding request and response is logged automatically so no need explicitly log that. You can see implementation in logging.go

Reply:
log messages are removed.